core/git_command.py

```diff
@@ -346,16 +342,9 @@ def git_binary_path(self):
 
             match = re.match(r"git version ([0-9]+)\.([0-9]+)\.([0-9]+)", stdout)
             if match:
-                major = int(match.group(1))
-                minor = int(match.group(2))
-                patch = int(match.group(3))
-                if major < GIT_REQUIRE_MAJOR \
-                        or (major == GIT_REQUIRE_MAJOR and minor < GIT_REQUIRE_MINOR) \
-                        or (major == GIT_REQUIRE_MAJOR and minor == GIT_REQUIRE_MINOR and patch < GIT_REQUIRE_PATCH):
-                    msg = GIT_TOO_OLD_MSG.format(
-                        GIT_REQUIRE_MAJOR,
-                        GIT_REQUIRE_MINOR,
-                        GIT_REQUIRE_PATCH)
+                version = tuple(map(int, match.groups()))
+                if version < MIN_GIT_VERSION:
```

Comment:
Could be useful to print out the found version to console here?

Reply:
Not enough fantasy right now for the correct wording, and how to ensure that this logging info is only printed once.

I think, it should show the the used `git_path` with its version. (Also maybe print on startup the used GitSavvy version. SublimeLinter does that.)